### PR TITLE
Issue #3265219 by chr.fritsch, volkerk: Fix coding style issues and achieve phpstan lvl6

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: tests/src/Kernel/Element/Select2Test.php [hunk 1]
```diff
@@ -89,9 +89,9 @@ public function testAutocompleteOptions() {
    * @param string $value
    *   The value of the option.
    */
-  protected function assertOptionExists($selector, $value) {
+  protected function assertOptionExists(string $selector, string $value): void {
     $select = $this->xpath('//select[@data-drupal-selector="' . $selector . '"]/option[@value="' . $value . '"]');
-    $this->assertEquals(1, count($select));
+    $this->assertEquals(1, is_countable($select) ? count($select) : 0);
```
Comment (chrfritsch) [inline tests/src/Kernel/Element/Select2Test.php]: use assertCount

File: tests/src/Kernel/Element/Select2Test.php [hunk 2]
```diff
@@ -102,9 +102,9 @@ protected function assertOptionExists($selector, $value) {
    * @param string $value
    *   The value of the option.
    */
-  protected function assertOptionNotExists($selector, $value) {
+  protected function assertOptionNotExists(string $selector, string $value): void {
     $select = $this->xpath('//select[@data-drupal-selector="' . $selector . '"]/option[@value="' . $value . '"]');
-    $this->assertEquals(0, count($select));
+    $this->assertEquals(0, is_countable($select) ? count($select) : 0);
```
Comment (chrfritsch) [inline tests/src/Kernel/Element/Select2Test.php]: use assertCount

File: tests/src/Kernel/Select2ValidOptionsTest.php [hunk 3]
```diff
@@ -39,7 +39,7 @@ public function testAvailableOptions() {
     $ref2->save();
 
     // Create a new revision to trigger problem.
-    $ref1->setName('Drupal')->setNewRevision();
+    $ref1->setName(\Drupal::class)->setNewRevision();
```
Comment (chrfritsch) [inline tests/src/Kernel/Select2ValidOptionsTest.php]: Revert this

**Review by ol0lll — APPROVED** [review]
lgtm

**Pull request merged**